client/package.json

```diff
@@ -4,10 +4,11 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^4.1.3",
+    "flag-icon-css": "^3.3.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
```

Comment:
The comma after "react-scripts" seems to be crashing the program on "npm start" for me.

Reply:
Thanks Eric! I'll fix that comma asap.